frontend/src/app/main/ui/workspace/header.cljs

```diff
@@ -362,14 +363,7 @@
         on-add-shared
         (mf/use-fn
          (mf/deps file-name add-shared-fn)
-         #(modal/show! {:type :confirm
-                        :message ""
-                        :title (tr "modals.add-shared-confirm.message" file-name)
-                        :hint (tr "modals.add-shared-confirm.hint")
-                        :cancel-label :omit
-                        :accept-label (tr "modals.add-shared-confirm.accept")
-                        :accept-style :primary
-                        :on-accept add-shared-fn}))
+         #(st/emit! (show-shared-dialog (:id file) add-shared-fn)))
```

Comment:
You are using `(:id file)` inside clusure, but you are not declaring it on the deps, consider using `file-id` on the deps, and seems like `file-name` on deps is not used anymore necessary